### PR TITLE
*: return the number of deleted keys

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/4576.

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @xiang90 @heyitsanthony 

Instead of `n`, I used `Deleted`, which I think is clearer. Please review. Thanks.

File: etcdserver/etcdserverpb/rpc.proto [hunk 1]
```diff
@@ -152,6 +152,7 @@ message DeleteRangeRequest {
 
 message DeleteRangeResponse {
   ResponseHeader header = 1;
+  int64 deleted = 2;
```
Comment (xiang90) [inline etcdserver/etcdserverpb/rpc.proto]: add comment?

**Comment by xiang90** [comment]
LGTM

**Comment by gyuho** [comment]
Thanks. Will merge in green lights (added comments).

**Pull request merged**